resources/local/config.toml

```diff
@@ -127,33 +127,20 @@ event_stream_buffer_length = 100
 # If unset, the path must be supplied as an argument via the CLI.
 path = '../node-storage'
 
-# Optional maximum size of the database to use for the block store.
-#
-# If unset, defaults to 483,183,820,800 == 450 GiB.
+# Maximum size of the database to use for the block store.
 #
 # The size should be a multiple of the OS page size.
-#max_block_store_size = 483183820800
+max_block_store_size = 483183820800
```

Comment:
Could you add the underscore separators to these values and also add back in the "human-readable" form for each value in the comments?  (Applies to the three values across the two config files)

Reply:
Wait, TOML supports these?